blocks/address/src/app.tsx

```diff
@@ -516,10 +544,21 @@ export const App: BlockComponent<RootEntity> = ({
                       }
                     }}
                     filterOptions={(options) => options}
+                    /**
+                     * By default, the anchor element for an autocomplete dropdown is the
+                     * input base, but we some uses of this component depend on resizing the
+                     * autocomplete root in order to attach the popup in a slightly different
+                     * place, so we make the autocomplete root the anchor element for the
+                     * popup.
+                     *
+                     * @see LinkEntityTypeSelector
+                     */
+                    ref={setAnchorEl}
                     renderInput={({ InputProps, ...params }) => {
                       return (
                         <TextField
                           {...params}
+                          autoFocus
```

Comment:
I just want to double check, do we actually want this? Wouldn't this mean that if this block were in a page with other blocks, there'd be a race between which block gets focused first?

Reply:
Yeah, thanks for checking on this. It's the desired behaviour, I think ([slack thread](https://hashintel.slack.com/archives/C03FRBNFM7T/p1677688755039609?thread_ts=1677688731.958359&cid=C03FRBNFM7T)). The openAI blocks do it with a `useEffect` that focuses the input

Reply:
Interesting, have we thought about the implications of having multiple blocks with autofocus inputs in the same page? If not perhaps worth flagging in that thread. Won't block the PR on this.